Files/Scripts/Options/Majora's Mask.psm1

```diff
@@ -2448,13 +2473,14 @@ function CreateTabEquipment() {
     # HITBOX #
 
     CreateReduxGroup  -Tag  "Equipment" -Text "Sliders"
-    CreateReduxSlider -Name "KokiriSword"      -Default 3000 -Min 512 -Max 8192 -Freq 512 -Small 256 -Large 512 -Text "Kokiri Sword"        -Info "Set the length of the hitbox of the Kokiri Sword"           -Credits "Admentus"
```

Comment:
Deku Stick change addition is good, but do not change anything else.

Reply:
Thought it would be good to change the description a little because I found a more clearer sounding way to say the sentence that was originally "Set the length of the hitbox of the (sword name here)" because "of the" was said twice and while it's possible that could be needed and not changed, I found a better way to write that which was "Set the hitbox length of the (sword name here)" which sounded better for me and reduced the spaces a little but it's my opinion. If you don't want that, I accept your choice.